styles.css

```diff
@@ -55,17 +55,33 @@
     margin: 0.75rem;
 }
 
-.introClose h1 {
+.instructionTop h1 {
     font-size: 40pt;
     margin-bottom: 0.5rem;
 }
 
-.introClose {
+.instructionTop {
     display: flex;
     justify-content: space-between;
     padding: 0;
 }
 
+.gameOverTop {
+    display: flex;
+    justify-content: space-between;
+    padding: 0;
+}
+
+.gameOverBtns {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.leftX {
+    width: 30px;
+}
```

Comment:
this class confuses me? it doesnt appear to do anything to do with being on the left or X?